cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/server.go

```diff
@@ -214,14 +232,16 @@ func (s *Server) GetPackageMeta(ctx context.Context, request *corev1.GetAvailabl
 	log.Infof("Found chart url: [%s]", *url)
 
 	// unzip and untar .tgz file
-	meta, err := fetchMetaFromChartTarball(request.AvailablePackageRef.Identifier, *url)
+	// TODO: userAgent, authz and netClient w/TLS config similar to asset-syncer utils.initNetClient(),
```

Comment:
If you can add your gh nick to the todo (ie. `// TODO (gfichtenholz): ...`) it'd be helpful (as the obvious committer from git is a pain to dig for as soon as this is refactored and copied elsewhere). Same for other TODOs in this change.

Reply:
will do